### PR TITLE
Update eeprom-guide.md

### PR DESCRIPTION
**Pull request by V205Github**
## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)
- Added Uno r4 boards.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/learn/03.programming/07.eeprom-guide/eeprom-guide.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ Functions in the **EEPROM** class is automatically included with the platform fo
 ## Hardware Required
 
 All of the following boards have an EEPROM:
-
+- [Arduino UNO Rev.4 Minima](https://store-usa.arduino.cc/products/uno-r4-minima)
```
Comment (jacobhylen) [inline content/learn/03.programming/07.eeprom-guide/eeprom-guide.md:14]:
```suggestion
- [Arduino UNO R4 Minima](https://store-usa.arduino.cc/products/uno-r4-minima)
```

File: content/learn/03.programming/07.eeprom-guide/eeprom-guide.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ Functions in the **EEPROM** class is automatically included with the platform fo
...
+- [Arduino UNO Rev.4 WiFi](https://store-usa.arduino.cc/products/uno-r4-wifi)
```
Comment (jacobhylen) [inline content/learn/03.programming/07.eeprom-guide/eeprom-guide.md:15]:
```suggestion
- [Arduino UNO R4 WiFi](https://store-usa.arduino.cc/products/uno-r4-wifi)
```

File: content/learn/03.programming/07.eeprom-guide/eeprom-guide.md [hunk 3]
```diff
@@ -11,7 +11,8 @@ Functions in the **EEPROM** class is automatically included with the platform fo
 ## Hardware Required
 
 All of the following boards have an EEPROM:
-
+- [Arduino UNO R4 Minima](https://store-usa.arduino.cc/products/uno-r4-minima)
```
Comment (jacobhylen) [inline content/learn/03.programming/07.eeprom-guide/eeprom-guide.md:14]:
```suggestion
- [Arduino UNO R4 Minima](https://store.arduino.cc/products/uno-r4-minima)
```

File: content/learn/03.programming/07.eeprom-guide/eeprom-guide.md [hunk 4]
```diff
@@ -11,7 +11,8 @@ Functions in the **EEPROM** class is automatically included with the platform fo
...
+- [Arduino UNO R4 WiFi](https://store-usa.arduino.cc/products/uno-r4-wifi)
```
Comment (jacobhylen) [inline content/learn/03.programming/07.eeprom-guide/eeprom-guide.md:15]:
```suggestion
- [Arduino UNO R4 WiFi](https://store.arduino.cc/products/uno-r4-wifi)
```

**Review by jacobhylen — APPROVED** [review]
Hi @V205Github :) 

I noticed the other EEPROM issue you added to our repo, and linked it. 
While you are correct, we've got this more extensive information in the dedicated EEPROM guide, and we're trying to keep the Cheat Sheets lean. 

Your contribution in this PR is another story, this is an important change and we're grateful you're taking the time to help us out with this 🙏

**Comment by V205Github** [comment]
Oh. Just read your comment. I might add that info to the dedicated EEPROM guide for the r4.
Thank you! @jacobhylen

**Pull request merged**